### DISCUSSION TITLE
Feature: Train Length Cheat Does Not Max Out Trains And Cars

### DISCUSSION THREAD

**Discussion by xp4xbox** [OpenRCT2/OpenRCT2#15722] [Feature requests]
An annoying thing is when you enable the train length cheat and go open a ride, it changes the trains and cars to be the max amount instead of default amount.

tagging @X123M3-256

**Comment by blackhand1001** [comment]
I am taking a look for the code for this right now. I am gonna try to make it default to the max size that will fit in the station and default to 1 less train than the number of blocks. 

That will be an improvement over the current 31 train default etc. I am just trying to track down when exactly it originally calculates that value so I can make it use that when picking the initial length while also allowing the user to increase it higher.

**Comment by xp4xbox** [comment]
Any update @blackhand1001 ?
